FirebaseRemoteConfig/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v4.4.7
+- [fixed] Fixed a crash that could occur when attempting a remote config fetch before a valid Instance ID was available. (#4622)
+- [fixed] Fixed an issue where config fetch would sometimes fail with a duplicate fetch error when no other fetches were in progress. (#3802)
+
```

Comment:
Thanks. Please also add 
- [changed] Fetch call will now fail if a valid instanceID is not obtained by the Remote Config SDK.

Reply:
Done.